### DISCUSSION TITLE
Authentication feature

### DISCUSSION THREAD

**Discussion by silvioprog** [deta/cloud-docs#594] [Ideas]
Hey team.

Some services similar to Deta provides an interesting authentication mechanism out-of-the-box, allowing us to implement authentication in our applications. For example:

- [supabase.com/auth](https://supabase.com/auth)

It would be awesome to have such a feature in Data! 😄 

Thank you very much!

**Comment by AyushSehrawat** [comment]
Well, they should look into it soon.. Also their drive is also good ( public private with links for public)

**Comment by nonan-io** [comment]
I asked the same thing [here](https://github.com/orgs/deta/discussions/564) a few weeks ago . It was mentioned as "coming soon" in the docs. Since then it has been removed from the docs, which is why I believe it's not on the roadmap right now. There is an auth feature coming with Deta Space. However, I'm still not convinced that Deta Space is the service I will use in the future. I don't understand why Deta doesn't think of a business model similar to Supabase.

Once you get access to Space, you can share your thoughts [here](https://github.com/orgs/deta/discussions/575).

**Reply by AyushSehrawat** [reply]
They will implement it in the next year probs since they are first trying to migrate to space... Also they are constantly making updates like base and all....

Yeah, i also suggested them supabase thingy, like their drive ( public and private bucket)

**Reply by pietz** [reply]
thanks for the info!

Deta seemed like the best thing since sliced bread during the first half of my project. Honestly. I had so little overhead for dealing with stuff I don't want to deal with. The closer I get to actually bringing a product live, the more I realize that I probably don't want to do it with Deta. It's a little sad. What I need is basically two things:

- Authorization as a Service
- Paid tier to upgrade certain resources

I'll keep my eyes open for Deta Space. For the moment, I'm a little let down that they focus on a service that will be hit or miss.
